README.md

```diff
@@ -197,3 +197,7 @@ This library requires C++11 or later. But if you're interested in a C or C++98
 API contact us on [gitter](https://gitter.im/opentracing/opentracing-cpp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
 We're open to supporting additional APIs in a separate repository if there are
 people willing to maintain it.
+
+## License
+
+By contributing to opentracing.cpp, you agree that your contributions will be licensed under its [MIT License](https://github.com/Shruti9520/opentracing-cpp/blob/master/LICENSE).
```

Comment:
Thanks, but can you change the link to https://github.com/opentracing/opentracing-cpp/blob/master/LICENSE

Reply:
Sorry. I did that by mistake. 
Thanks for reviewing. :-)